### PR TITLE
Update story northern union.txt

### PR DESCRIPTION
**Pull request by TheMarksman-ES**
NOTICE: Delete the sections that do not apply to your PR, and fill out the section that does.
(You can open a PR to add or improve a section, if you find them lacking!) 

----------------------
**Content (Artwork / Missions / Jobs)**

## Summary
This was a lot easier to edit. Overall, it's an interesting mission string.

## Fix Details
Edits and Choice word changes.

### CODE REVIEW

File: data/story northern union.txt [hunk 1]
```diff
@@ -31,7 +31,7 @@ mission "Meeting an Alpha 0"
 	
 	on offer
 		conversation
-			`A man in a tattered and heavily defaced Navy uniform approaches you and asks whether you would be willing to give him a lift to <destination>. The jacket has "Warlord Peg" stitched where the name patch normally would be.`
+			`A man in a tattered and heavily defaced Navy uniform approaches you as you walked out of your ship and asking whether you would be willing to give him a lift to <destination>. His jacket has "Warlord Peg" stitched where the name patch normally would be.`
```
Comment (Lorantine) [inline data/story northern union.txt:34]: This is wrong grammar-wise.
Change this:

> A man in a tattered and heavily defaced Navy uniform approaches you as you walked out of your ship and asking whether you would be willing to give him a lift to \<destination\>.

To this:

> A man in a tattered and heavily defaced Navy uniform approaches you as you walk out of your ship and asks whether you would be willing to give him a lift to \<destination\>.

File: data/story northern union.txt [hunk 2]
```diff
@@ -56,7 +56,7 @@ mission "Meeting an Alpha 0"
 	on complete
 		payment
 		dialog
-			`The warlord thanks you for the ride and pays you <payment>. As he is leaving, he says, "If you're interested in meeting my boss, head to a shop called "The Rusted Wheel" and find my commander. He's impossible to miss, try not to stare."`
+			`The warlord thanks you for the ride and pays you <payment>. As he is leaving, he says, "If you're interested in meeting my boss, head to a shop called "The Rusted Wheel" and find my commander. He's impossible to miss, though, try not to stare."`
```
Comment (Lorantine) [inline data/story northern union.txt:59]: I think with a comma might also be correct, but I'd prefer without the comma.
Why? This way to me it reads as if "though" is part of the first part: "He's impossible to miss, though."
While it belongs to the last part: "though he's impossible to miss."

File: data/story northern union.txt [hunk 3]
```diff
@@ -82,9 +82,12 @@ mission "Meeting an Alpha 1"
...
+			`	"Good decision, joining me. My boss will be here any minute now."` 
+			
+			`Some time passes before eventually an incredibly tall, noseless man joins you. He looks almost exactly like the descriptions of the Augments known as Alphas from centuries ago.`
+			
+			`	"Boss, I've brought you a tribute!" Peg shouts, as you feel a hard, cold object press against the side of your skull. Before you even get a chance to react, you feel something warm splattering on your cheek. You look over to see Peg, dying, with some sort of metal spike sticking out of his forehead.`
+			`	"I would appologize for the mess," the towering beast drones, "Considering I just saved your life from my incompetent underling, I'll assume you've already forgiven me." He gives off a strange huffing sound that almost seems like a chuckle, "Please, stay. I sense you aren't as useless as this deflating mess next to you."`
```
Comment (Lorantine) [inline data/story northern union.txt:90]: Change:

> appologize

To:

> apologize

Comment (Lorantine) [reply data/story northern union.txt:90]: Make the capital C a normal c in "Considering", as it is a continuation of the earlier quoted text, not a new sentence.

File: data/story northern union.txt [hunk 4]
```diff
@@ -139,9 +142,9 @@ mission "Meeting an Alpha 1"
...
+			`	"I'm sure you know already that we were created as super soldiers to be used by various factions, but the reason for that is because those in charge were getting reports of normal soldiers refusing orders.  Things such as leveling towns full of innocent bystanders, shooting down unarmed passenger ships, contaminating spaceports with radiation or bioweapons, that sort of thing. The human soldiers were apparently brought up with the idea that 'life was sacred' or something." His tone turns to what you can only assume is mockery. "And that 'only bad people' should be killed. And when faced with the flimsy excuses for why they were slaughtering these innocents, they tended to refuse like the cowards they were."`
 			`	The being calls over a waiter, and whispers his order, before continuing. There is an interesting smell in the air, which both turns your stomach and compels you to be silent.`
```
Comment (Lorantine) [inline data/story northern union.txt:146]: Remove the double space before "Things".

Comment (Lorantine) [reply data/story northern union.txt:146]: Change:

> I'm sure you know already

To:

> I'm sure you already know

Comment (Lorantine) [reply data/story northern union.txt:146]: Change:

> that sort of thing

To:

> that sort of things

Since before the "things" talked about are plural, and there's multiple things,

Comment (Lorantine) [reply data/story northern union.txt:146]: Change:

> like the cowards they were.

To:

> like the cowards they are.

In this context, I'd imagine the one calling them cowards believes they still are cowards, so it should be present, not past.

File: data/story northern union.txt [hunk 5]
```diff
@@ -139,9 +142,9 @@ mission "Meeting an Alpha 1"
...
+			`	"So, where was I?" He pauses, "Ah yes, our origins. It was decided that there was a need for soldiers who were "emotionless," and, well, cheaper to maintain for increased combat efficiency. Starting with your "basic" human, it was decided to focus on developing a streamlined combat being, one which would obey orders without question, and could be fielded without much overhead. Turns out your human frame is about the only useful thing about you in combat, however. The limits of genetic engineering at the time, however, seemed to be infinite. So they started taking samples from various bugs. Cockroaches, bacteria strains, grasshoppers. They set out to create the perfect beings, us. A perfect blend of strengh, agility, tactical prowess, radiation and bioweapon resistances, and natural built in and regenerating armor in the form of our exoskeleton. Capable of surviving and fighting for hours in empty space, or under water, or in most any environment, really. Only occasionally with extra gear."`
```
Comment (Lorantine) [inline data/story northern union.txt:147]: Change:

> Turns out your human frame is about the only useful thing about you in combat, however. The limits of genetic engineering at the time, however, seemed to be infinite. So they started taking samples from various bugs.

To:

> Turns out your human frame is the only useful thing about you in combat. However, the limits of genetic engineering at the time, seemed to be infinite, so they started taking samples from various bugs.

The double "about" in 1 sentence reads awkward, and 2 instances of "however" in 2 consecutive sentences even more so. Lastly, the last sentence in my quote reads better when part of the previous one, since starting a sentence with "so" is generally a thing to avoid if possible (same with words like "but").

Comment (Lorantine) [reply data/story northern union.txt:147]: Change:

> Cockroaches, bacteria strains, grasshoppers. They set out to create the perfect beings, us.

To:

> Cockroaches, bacteria strains, grasshoppers, they set out to create the perfect beings, us.

To be honest, I'm not sure if this is the best "fix", but having a sentence which only consists of a few "species" words doesn't read that nice.
Alternatively, you could instead merge the first quoted sentence with the sentence before it instead, and leave the second quoted sentence alone.

File: data/story northern union.txt [hunk 6]
```diff
@@ -198,7 +201,7 @@ mission "Meeting an Alpha 1"
 			`	You cut the commlink, relieved that it was that easy.`
 	
 	on complete
-		dialog `Soon after you land, a courier comes to your ship and pays you <payment> in exchange for your sensor logs regarding the <npc>. She says, "If you would be interested in continuing to work for my Godlord, render yourself before him at the Rusted Wheel Cafe. He has a gift for you."`
+		dialog `Soon after you land, a courier comes to your ship and pays you <payment> in exchange for your sensor logs regarding the <npc>. "If you would be interested in continuing to work for my Godlord, render yourself before him at the Rusted Wheel Cafe. He has a gift for you."`
```
Comment (Lorantine) [inline data/story northern union.txt:204]: I think it's better to revert this change. Having a quote from a courier on its own without using either '[person] says, "Quote."' or '"Quote," [person] says.' reads a bit awkward, and I'm not sure if this is even correct grammar.

**Comment by Lorantine** [comment]
Some of the things are pointed out were already wrong before, but correcting them would be appreciated. I think if you incorporate the feedback this should be merge-ready.

**Comment by kikotheexile** [comment]
A huge thank you to @Lorantine for pointing out some additional changes that need made. This will, of course, be merged initially without the fixes, but the fixes will be included in later iterations done by myself or one of yourselves. 

But progress is progress for now, and it's better to keep it marching forward than to stall it.
